cg/cli/workflow/rnafusion/base.py

```diff
@@ -42,3 +50,56 @@ def rnafusion(context: click.Context) -> None:
 rnafusion.add_command(store_housekeeper)
 rnafusion.add_command(store)
 rnafusion.add_command(store_available)
+
+
+@rnafusion.command()
+@ARGUMENT_CASE_ID
+@click.pass_obj
+def dev_config_case(config: CGConfig, case_id: str):
+    """Configure an RNAFusion case so that it is ready to be run."""
+    factory = ConfiguratorFactory(config)
+    configurator: NextflowConfigurator = cast(
+        NextflowConfigurator, factory.get_configurator(Workflow.RNAFUSION)
+    )
+    configurator.configure(case_id=case_id)
+
+
+@rnafusion.command()
+@ARGUMENT_CASE_ID
+@OPTION_REVISION
+@OPTION_STUB
+@click.pass_obj
+def dev_run(config: CGConfig, case_id: str, stub_run: bool = False, revision: str | None = None):
```

Comment:
According to what we just discussed, it seems that the order of the decorators does not coincide with the order of the arguments. Resolve if I'm wrong
```suggestion
@OPTION_REVISION
@OPTION_STUB
@ARGUMENT_CASE_ID
@click.pass_obj
def dev_run(config: CGConfig, case_id: str, stub_run: bool = False, revision: str | None = None):
```

Reply:
I would need to test it! It might be that it is only the passed object that is treated separately. Let me get back to you on that!